app/templates/app/projects/create.hbs

```diff
@@ -97,7 +97,7 @@
                   @aclContext="App.Project.Create.EndDate"                            
                 />
 
-                <FormFields::FieldRelate
+                {{!-- <FormFields::FieldRelate
```

Comment:
Instead of commenting the field, just remove it since these changes are being maintained by the GIT repo.